src/components/CreateToken.vue

```diff
@@ -1,75 +1,136 @@
 <template>
-  <div>
-    <!-- eslint-disable vue-i18n/no-raw-text -->
+  <div class="create-token">
     <div class="create-header">
       <div class="create-header-content">
-        <h2>Create Your Token</h2>
-        <h3>In Less Than 5 Minutes</h3>
-        <span class="step-description">{{ stepDescription }}</span>
+        <h2>
+          {{ success ?
+            $t('components.CreateToken.NewToken') :
+            $t('components.CreateToken.CreateToken') }}
+        </h2>
+        <h3>
+          {{ success ?
+            `Redirecting in ${seconds} seconds` :
+            $t('components.CreateToken.FiveMinutes') }}
+        </h3>
+        <i18n
+          :path="`components.CreateToken.StepsDescription[${step}]`"
+          tag="p"
+          class="step-description"
+        >
+          <span class="abbreviation">{{ ticker }}</span>
+        </i18n>
       </div>
+      <div class="arrow-up" />
     </div>
-    <div class="steps">
-      <div
-        v-for="i in 6"
-        :key="i"
-        class="step"
-      >
+    <div class="steps-wrapper">
+      <div class="steps">
         <div
-          class="step-box"
-          :class="{ active: step >= i, pulse: (step === i && step !== 1) }"
-        />
-      </div>
-    </div>
-    <div class="create-inputs input-group">
-      <div>
-        <label for="name">Tokenized asset</label>
-        <input
-          id="name"
-          v-model="name"
-          placeholder="Enter any combination of characters *"
-          class="form-control"
-          minlength="1"
-          maxlength="333"
-          :disabled="loadingState"
-        >
-      </div>
-
-      <div>
-        <label for="description">Token Description</label>
-        <input
-          id="description"
-          v-model="description"
-          placeholder="Enter additional information about your token or community *"
-          class="form-control"
-          minlength="1"
-          maxlength="500"
-          :disabled="loadingState"
+          v-for="i in 6"
+          :key="i"
+          class="step"
         >
+          <div
+            class="step-box"
+            :class="{ active: step >= i - 1, pulse: ((step === i - 1 && step !== 0) || step > 6) }"
```

Comment:
this looks quite complicated, what was the issue with the previous implementation?